AB_environments/config.yaml

```diff
@@ -14,3 +14,16 @@ categories:
   - benchmarks
   # - runtime
   # - stability
+
+# AWS implements limiters to how many EC2 instances you can spawn in parallel on the
+# same AWS account. If such limit is reached, tests will randomly fail when trying to
+# create the Coiled clusters, and restarting failed jobs won't fix the problem.
+# Additionally, there are problems with Coiled itself triggered by limitations that are
+# never actually reached with real paying users.
+max_parallel:
+  # Number of parallel A/B test jobs per branch.
+  ci_jobs: 5
+  # Number of parallel test_*.py modules per A/B test job.
+  # Each module typically spawns one Coiled cluster at a time.
+  # Set to 1 to disable pytest-xdist.
+  modules_per_job: 4
```

Comment:
nit: I first ran into the variable name without the description and it took me a moment to figure out how the number of modules would impact parallelism. Would it make sense to rename this to something like `pytest_workers_per_job` to highlight that this is about `pytest-xdist` parallelism?

Reply:
done